modules/argocd-helm/variables.tf

```diff
@@ -51,13 +51,45 @@ variable "oidc" {
   }
 }
 
+variable "argocd" {
+  description = "ArgoCD settings"
+  type        = object({
+    domain = string
+  })
```

Comment:
Shouldn't we add also `scheme` and `port` or just pass the `url` and extract `scheme`, `hostname` and `port` from it?

Reply:
I think for now assuming https & 443 is reasonable.